appveyor.yml

```diff
@@ -15,9 +15,9 @@ cache:
 
 install:
 - ps: >-
-    (new-object net.webclient).DownloadFile('https://download.microsoft.com/download/A/2/D/A2D8587D-0027-4217-9DAD-38AFDB0A177E/msxml.msi', 'C:\\msxml.msi')
+    (new-object net.webclient).DownloadFile('https://download.microsoft.com/download/a/4/5/a45cb3d5-2268-4d13-9efb-ec800262ac98/msxml6.msi', 'C:\\msxml6.msi')
```

Comment:
This is a 32 bit version of msxml6.
Once we create a 64 bit version of SimModel we might also require a 64 bit version of msxml6
https://download.microsoft.com/download/a/4/5/a45cb3d5-2268-4d13-9efb-ec800262ac98/msxml6_x64.msi